circt.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+GIT_COMMIT=093cdfe482530623fea01e1d3242af93e533ba54
+
+CIRCT_BUILD=./build-circt
```

Comment:
The script should take the `CIRCT_PATH` as argument and set it. You are using this build path in the CI workflows, but it is implicitly hidden here.

Reply:
Added `--build-path` and `--install-path` to the script such that they can be set manually if one want to place it somewhere else than the default location.

Reply:
I added explicit paths in the action and workflow. Not sure if that's what you wanted.
The drawback is that any changes to the PATHS in build-circt.sh and configure.sh now requires the CI to also be updated.